src/components/View/index.ts

```diff
@@ -120,7 +120,7 @@ class View extends BridgeBase {
     }
     this.title = 'title' in configProps ? configProps.title : '';
 
-    const inheritedProps$ = this.thePConn.getInheritedProps();
+    const inheritedProps$: any = this.thePConn.getInheritedProps(); // try to remove any when getInheritedProps typedefs are fixed
```

Comment:
Can you feed this back to Optimus please.

Reply:
sure Niall